ebu_tt_live/node/delay.py

```diff
@@ -2,8 +2,9 @@
 import six
 import logging
 from .base import AbstractCombinedNode
-from datetime import timedelta
+from datetime import timedelta, datetime
```

Comment:
Is `from datetime import datetime` different? It won't import `timedelta` as well?